Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
 	"frame/office",
```

Comment:
Add you pallet to our github workflows (I remember at least `cargo fmt --check`)